content/user/vulns/ManageVulns.md

```diff
@@ -26,6 +26,32 @@ Contrast provides multiple ways to narrow findings and focus on the vulnerabilit
 
 <a href="assets/images/Vulnerability_Filters.png" rel="lightbox" title="Vulnerability Filters"><img class="thumbnail" src="assets/images/Vulnerability_Filters.png"/></a>
 
+## Pending Vulnerabilities 
+
+If an administrator [requires administrative approval](admin-orgsettings.html#app-defaults) to close vulnerabilities, vulnerabilities with their specified statuses and severities go into a **Pending** state when you attempt to close them. Administrative approval applies to changes to vulnerabilities by two-way [bugtracker integrations](admin-orgintegrations.html#bugtracker) as well as [auto-remediation policies](admin-policymgmt.html#remediate).  
+
+When you attempt to close a vulnerability that an administrator must approve, you must provide a reason and justification for the status change. Once completed, Contrast confirms that the closure is pending an administrator's review. 
+
+<a href="assets/images/Vulns-mark-as-dialog.png" rel="lightbox" title="Enter a reason and justification for a status change"><img class="thumbnail" src="assets/images/Vulns-mark-as-dialog.png"/></a>
+
+Once Contrast confirms your request, the vulnerability is marked as **Pending** in the grid. Hover over the label to see more information about when the request to close was submitted. To see all vulnerabilities awaiting review in your organization, select the **Pending Review** filter from the dropdown. 
```

Comment:
We refer to these as "quick views" since they don't always correlate directly to a filter (i.e. "Policy Violation")